packages/config/src/doc/IProfArgAttrs.ts

```diff
@@ -11,6 +11,12 @@
 
 import { IProfLoc } from "./IProfLoc";
 
+/** The type of data for this property */
+export type IProfDataType = "string" | "number" | "boolean" | "array" | "object";
+
+/** The value for the argument */
+export type IProfArgValue = string | number | boolean | string[] | object;
```

Comment:
Thanks for making this a separate type 🙂 Curious why we chose `string[]` rather than `any[]` for the array type here. Is it because the CLI always parses each item in an array arguments as a string?

Reply:
I believe so, but will change it to `any[]` to be safe here

Reply:
Is changing this potentially a v2-v2 breaking change? For anyone who was depending on the type being `string[]`

Reply:
After looking into this a bit more...
Its main use was for getting arrays from ENV variables. so they will always be strings.
So, maybe it's fine to keep it as `string[]`